src/lib/utils.ts

```diff
@@ -45,3 +45,59 @@ export const validateValue = (val: Value): void => {
     `unexpected value type ${typeof val}, expected boolean | number | Temporal.Instant | string`
   );
 };
+
+export const convertStringToXsdValue = (dataType: string, valueStr: string): Value => {
+  let value: Value;
+  switch (dataType) {
+    case XSDNS.Boolean:
+      switch (valueStr) {
```

Comment:
is it possible to have number type here?

Reply:
no.